src/dialogs/more-info/controls/more-info-person.ts

```diff
@@ -34,7 +34,6 @@ class MoreInfoPerson extends LitElement {
         : ""}
       ${!__DEMO__ &&
       this.hass.user?.is_admin &&
-      this.stateObj.state === "not_home" &&
```

Comment:
We still don't want to offer the user to create a zone when they are home.

Reply:
Don't we? My "home" zone is nested within a "London" zone. What's different about home compared to others?

Reply:
Well yeah I guess that’s a use case